helm/charts/determined/templates/priority-classes.yaml

```diff
@@ -7,26 +7,26 @@ preemptionPolicy: Never
 globalDefault: false
 description: "This priority class should be used for Determined system pods only."
 ---
-{{- if .Values.defaultScheduler}}
-{{- $schedulerType := .Values.defaultScheduler | trim}}
-{{- if or (eq $schedulerType "coscheduler") (eq $schedulerType "preemption")}}
 apiVersion: scheduling.k8s.io/v1
 kind: PriorityClass
 metadata:
-  name: determined-high-priority
-value: 100
+  name: determined-medium-priority
+value: 50
 preemptionPolicy: Never
 globalDefault: false
-description: "This priority class should be used for high priority Determined jobs."
+description: "This priority class should be used for medium priority Determined jobs."
 ---
+{{- if .Values.defaultScheduler}}
+{{- $schedulerType := .Values.defaultScheduler | trim}}
+{{- if or (eq $schedulerType "coscheduler") (eq $schedulerType "preemption")}}
 apiVersion: scheduling.k8s.io/v1
 kind: PriorityClass
 metadata:
-  name: determined-medium-priority
-value: 50
+  name: determined-high-priority
```

Comment:
I don't understand why determined-high-priority is still here, if not for backwards compatibility, nor why it's now inside the conditional but medium isn't. Could you enlighten me?

Reply:
Good point, we actually don't need `determined-high-priority` or `determined-low-priority` anymore. I can just straight up remove them from the helm chart.